File: 3st/Juhee/문자열 정렬하기 (1).java
import java.util.Arrays;

class Solution {
    public int[] solution(String my_string) {
    	my_string = my_string.replaceAll("[a-z]", ""); 
        
        String[] arr = my_string.split("");
        int[]answer = new int[arr.length];
        
        for (int i = 0; i < arr.length; i++) {
        	answer[i] = Integer.parseInt(arr[i]);
		}
        Arrays.sort(answer);
        
    	return answer;
    }
}

File: 3st/Juram/programmer/문자열 정렬하기 (1).js
function solution(my_string) {
    let answer = [];
    let num = my_string.replace(/[^0-9]/g, "");
    for(let n of num){
        n = Number(n);
        answer.push(n);
    }
    answer.sort();
   
   
    return answer;
}